paperless-ngx update fails because /usr/local/bin isn't on the path and uv is not found

### ✅ Have you read and understood the above guidelines?

Yes

### 📜 What is the name of the script you are using?

paperless-ngx

### 📂 What was the exact command used to execute the script?

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/paperless-ngx.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

Tried updating paperless-ngx container today (via /bin/update), and it failed with the error:
```
/dev/fd/63: line 1340: uv: command not found

[ERROR] in line 1340: exit code 0: while executing command VERSION_MATCH=$(uv python list --only-downloads | grep -E "^cpython-${PYTHON_VERSION//./\.}\.[0-9]+-linux" | cut -d'-' -f2 | sort -V | tail -n1)
```

Looking into it, uv did install correctly, but it's installed in `/usr/local/bin`, which is not in `PATH` by default. Fixed it for me via `echo 'export PATH="$PATH:/usr/local/bin"' >> ~/.bashrc && source ~/.bashrc`, but might affect others with older installations as well, so thought I'd report it.


### 🔄 Steps to reproduce the issue.

- Have an old paperless-ngx installation where `/usr/local/bin` isn't on the path
- Try to `/bin/update`


### ❌ Paste the full error output (if available).

```
    ____                        __
   / __ \____ _____  ___  _____/ /__  __________      ____  ____ __  __
  / /_/ / __ `/ __ \/ _ \/ ___/ / _ \/ ___/ ___/_____/ __ \/ __ `/ |/_/
 / ____/ /_/ / /_/ /  __/ /  / /  __(__  |__  )_____/ / / / /_/ />  <
/_/    \__,_/ .___/\___/_/  /_/\___/____/____/     /_/ /_/\__, /_/|_|
           /_/                                           /____/

  ✔️   Stopped all Paperless-ngx Services
   💡   You are about to migrate your Paperless-ngx installation to uv!
   1. Stop the container:  pct stop <CTID>
   2. Create a snapshot:  pct snapshot <CTID> pre-paperless-uv-migration
   3. Start the container again

Have you created a snapshot? [y/N]: y
  ✔️   Patched paperless-webserver.service
  ✔️   Patched paperless-scheduler.service
  ✔️   Patched paperless-consumer.service
  ✔️   Patched paperless-task-queue.service
  ✔️   Backup completed
  ✔️   Setup uv 0.8.13
/dev/fd/63: line 1340: uv: command not found

[ERROR] in line 1340: exit code 0: while executing command VERSION_MATCH=$(uv python list --only-downloads | grep -E "^cpython-${PYTHON_VERSION//./\.}\.[0-9]+-linux" | cut -d'-' -f2 | sort -V | tail -n1)

root@paperless-ngx:~#
```

### 🖼️ Additional context (optional).

_No response_

--- a/misc/tools.func
+++ b/misc/tools.func
@@ -1328,12 +1328,13 @@ function setup_uv() {
     return 1
   }
 
-  rm -rf "$TMP_DIR"
-  #ensure_usr_local_bin_persist
-  $STD uv python update-shell || {
-    msg_error "Failed to update uv shell integration"
-    return 1
-  }
+  if [[ ":$PATH:" != *":/usr/local/bin:"* ]]; then
+    export PATH="/usr/local/bin:$PATH"
+  fi
+  ensure_usr_local_bin_persist
+  if ! $STD uv python update-shell; then
+    msg_error "uv shell integration failed – continuing anyway"
+  fi
   msg_ok "Setup uv $LATEST_VERSION"
 
   # Optional: install specific Python version

--- a/misc/tools.func
+++ b/misc/tools.func
@@ -1329,7 +1329,11 @@ function setup_uv() {
   }
 
   rm -rf "$TMP_DIR"
-  ensure_usr_local_bin_persist
+  #ensure_usr_local_bin_persist
+  $STD uv python update-shell || {
+    msg_error "Failed to update uv shell integration"
+    return 1
+  }
   msg_ok "Setup uv $LATEST_VERSION"
 
   # Optional: install specific Python version

